jersey-tck/src/test/resources/arquillian.xml

```diff
@@ -0,0 +1,15 @@
+
+<arquillian>
+<container qualifier="arq-glassfish-managed">
+    <configuration>
+        <property name="allowConnectingToRunningServer">false</property>
+        <!--            <property name="glassFishHome">target/${glassfish.home}</property>-->
```

Comment:
Please either remove, uncomment, or add description why it is commented.

Reply:
Have removed the file arquillian.xml for now as it is not used. I have set the environment variables in jersey-tck/pom.xml